### PR TITLE
Adds Europan clothes.

### PR DESCRIPTION
**Pull request by TheStryker**
Adds a Europan whalebone charm accessory, submariner coat, and induction lighter as part of the lore. Special thanks to the human lore team for the descriptions.

image
captain ramius

### CODE REVIEW

**Comment by TheStryker** [comment]
!review

File: code/modules/clothing/factions/europa.dm [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/obj/item/clothing/suit/storage/toggle/europa_coat
+	name = "Europan submariners’ coat"
```
Comment (Alberyk) [inline code/modules/clothing/factions/europa.dm:2]:
```suggestion
	name = "Europan submariners' coat"
```

File: code/modules/clothing/factions/europa.dm [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	desc = "A leather jacket with a synthetic fur collar. It looks comfortable, if a bit warm for the station."
+	desc_fluff = "Waterproof leather costs such as this one are commonly worn by Europan submarine crews due to their ability to keep you warm while keeping you dry. The synthetic fur on this one’s collar seems to have been imported from Callisto."
```
Comment (Alberyk) [inline code/modules/clothing/factions/europa.dm:4]:
```suggestion
	desc_fluff = "Waterproof leather costs such as this one are commonly worn by Europan submarine crews due to their ability to keep you warm while keeping you dry. The synthetic fur on this one's collar seems to have been imported from Callisto."
```

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 3]
```diff
@@ -290,3 +290,7 @@
 	display_name = "shoulder sleeve patch"
 	path = /obj/item/clothing/accessory/sleevepatch
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/whalebone
```
Comment (Alberyk) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm:294]: Need a to set this entry flags to GEAR_HAS_DESC_SELECTION

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 4]
```diff
@@ -363,3 +363,7 @@
 	coat["feminine chokha"] = /obj/item/clothing/suit/storage/vysoka_f
 	coat["masculine chokha"] = /obj/item/clothing/suit/storage/vysoka_m
 	gear_tweaks += new/datum/gear_tweak/path(coat)
+
+/datum/gear/suit/europa
```
Comment (Alberyk) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm:367]: Need a to set this entry flags to GEAR_HAS_DESC_SELECTION

**Review by Alberyk — CHANGES_REQUESTED** [review]

File: code/modules/clothing/factions/europa.dm (hunk 1 above)
Comment (TheStryker) [reply code/modules/clothing/factions/europa.dm]: Grammar errors fixed.

File: code/modules/clothing/factions/europa.dm (hunk 2 above)
Comment (TheStryker) [reply code/modules/clothing/factions/europa.dm]: Grammar errors fixed.

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm (hunk 3 above)
Comment (TheStryker) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm:294]: Added to this and the whalebone.

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
Please make the whale bone accessory smaller to be in line with the other accessories.
Something along the lines of the dog tags. Thanks.

**Comment by TheStryker** [comment]
> Please make the whale bone accessory smaller to be in line with the other accessories.
> Something along the lines of the dog tags. Thanks.

image
Jackets are dead,

**Review by alsoandanswer — APPROVED** [review]
Cool. I'll work on a recolorable jacket.

**Review by Geevies — APPROVED** [review]

**Review by Doxxmedearly — APPROVED** [review]

**Pull request merged**